tasks/hyrax-metadata-updates/tests/test-integration-index.js

```diff
@@ -80,25 +80,32 @@ test.before(async () => {
   }).promise();
 });
 
-test.beforeEach(() => {
+const setupNock = (params) => {
```

Comment:
Should this be setupMock?
```suggestion
const setupMock = (params) => {
```

Reply:
I don't think so, the function continues to L102 and sets up nock so that it responds correctly.

Reply:
nevermind! just learned about https://github.com/nock/nock